_summaries/sukhbaatar2015end-to-end_1.md

```diff
@@ -1,19 +1,76 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: sukhbaatar2015end-to-end
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: DartingMelody
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+**What is the core idea?**
+
+The paper introduces a novel recurrent neural network with an explicit large memory and a recurrent attention mechanism. The model reads recurrently from memory multiple times before giving the results. This model is trained end-to-end and therefore requires comparitively less supervision during training. Hence, it can also be applied to realistic settings. The paper also shows experimentally that multiple hops over the long-term memory leads to good performance of the model. 
+
+**How is it realized (technically)?**
```

Comment:
Before diving into how it is realized, a quick explanation of what a Q&A task looks like might be helpful.

I personally did not understand most of the article (like what is a xi really) until I saw the example of the Synthetic Question and Answering Experiments in section 4.  

By adding a small intro section to frame the model, your summary would really pop :D